### PR TITLE
[Docs] PR for Document Inspect & Modify flow

### PR DESCRIPTION
**Pull request by lizayugabyte**
The requested content added at the end of section https://deploy-preview-12724--infallible-bardeen-164bc9.netlify.app/preview/deploy/multi-dc/async-replication/#bootstrap-a-target-universe

### CODE REVIEW

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 1]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
-  ```
+- Create tables in both universes, as follows: 
 
-  (At target)
-  ```sh
-  kubectl exec -it -n <target_universe_namespace> -t <target_universe_master_leader> -c <target_universe_container> -- bash
-  /home/yugabyte/bin/ysqlsh -h <target_universe_yqlserver>
-  create table query
-  ```
+  - Execute the following commands at for the source universe:
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:280]: commands at for the

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 2]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
 
-  (At source)
-  ```sh
+      ```sh
+    kubectl exec -it -n xcluster-source -t yb-master-2 -c yb-master -- bash
+      /home/yugabyte/bin/ysqlsh -h yb-tserver-1.yb-tservers.xcluster-source
+      create table employees(id int primary key, name text);
+      ```
+
+  - Execute the following commands at for the target universe:
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:296]: commands at for the

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 3]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
-  ```
+    "/home/yugabyte/bin/yb-admin -master_addresses yb-master-2.yb-masters.xcluster-target.svc.cluster.local, \
+    yb-master-1.yb-masters.xcluster-target.svc.cluster.local,yb-master-0.yb-masters.xcluster-target.svc.cluster.local \
+    setup_universe_replication ac39666d-c183-45d3-945a-475452deac9f_xCluster_1 \
+    yb-master-2.yb-masters.xcluster-source.svc.cluster.local,yb-master-1.yb-masters.xcluster-source.svc.cluster.local, \
+    yb-master-0.yb-masters.xcluster-source.svc.cluster.local 00004000000030008000000000004001"
+    ```
 
-5. Perform some DMLs on the source side and observe the replication at the target side.
+- Perform the following on the source universe and then observe replication one the target universe:
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:334]:
```suggestion
- Perform the following on the source universe and then observe replication on the target universe:
```

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 4]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
+- To add a table to the source and target universes, use the `alter_universe_replication add_table` command.
+- To remove an existing table from the source and target universes, use the `alter_universe_replication remove_table` command.
+- To change the master nodes on the source universe, execute the `alter_universe_replication set_master_addresses` command.
+- You can verify changes via the `get_universe_config` command.
 
-## Schema migration
+## Migrate schema
 
-This section describs how to execute DDL operations, after replication has been already configured for some tables.
+You can execute DDL operations, after replication has been already configured for some tables.
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:429]: You can execute DDL operations, even after replication has been already configured for some tables.
or
You can execute DDL operations after replication has been already configured for some tables.
?

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 5]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
 
-In the event you cannot stop incoming user traffic, then the recommended approach would be to apply DDLs on the source cluster and use [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster) flow above. A more detailed runbook would be as follows:
-- Stop replication, in advance of DDL changes.
-- Apply all your DDL changes to the source cluster.
-- Take a backup of the source cluster, of all the relevant tables that you intend to replicate changes for.
-  - Make sure to use the [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster) flow, as described above.
-- Restore this backup on the sink cluster.
-- [Setup replication](../../../admin/yb-admin/#setup-universe-replication) again, for all of the relevant tables.
-  - Make sure to pass in the `bootstrap_ids`, as described above.
+If you cannot stop incoming user traffic, then the recommended approach would be to apply DDLs on the source universe combined with bootstrapping. Specifically, you would need to do the following:
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:441]:
```suggestion
If you can't stop incoming user traffic, then the recommended approach would be to apply DDLs on the source universe combined with bootstrapping. Specifically, you would need to do the following:
```

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 6]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
+- Stop replication before making any DDL changes.
+- Apply all your DDL changes to the source universe.
+- Backup the source universe and all the relevant tables for which you intend to replicate changes. Follow instructions provided in [Bootstrap a target universe](#bootstrap-a-target-universe).
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication.md:444]:
```suggestion
- Back up the source universe and all the relevant tables for which you intend to replicate changes. Follow instructions provided in [Bootstrap a target universe](#bootstrap-a-target-universe).
```

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 7]
```diff
@@ -158,23 +158,21 @@ For example:
...
 
-1. For each master and tserver on the target universe, set the gflag `certs_for_cdc_dir` to the parent directory where you will store all the source universe's certs for replication.
+1. Find the certificate authority file used by the source universe (`ca.crt`). This should be stored within the [`--certs_dir`]`/preview/reference/configuration/yb-master/#certs-dir`.
 
-1. Find the certificate authority file used by the source universe (`ca.crt`). This should be stored within the [`--certs_dir`]/preview/reference/configuration/yb-master/#certs-dir).
+1. Copy this file to each node on the target. It needs to be copied to a directory named`<certs_for_cdc_dir>/<source_universe_uuid>`.
 
-1. Copy this file to each node on the target. It needs to be copied to a directory named: `<certs_for_cdc_dir>/<source_universe_uuid>`.
+    For example, if you previously set `certs_for_cdc_dir=/home/yugabyte/yugabyte_producer_certs`, and the source universe's ID is `00000000-1111-2222-3333-444444444444`, then you would need to copy the certificate file to `/home/yugabyte/yugabyte_producer_certs/00000000-1111-2222-3333-444444444444/ca.crt`.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:171]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 8]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
-## Setting up Replication with Geo-partitioning
+## Set up replication with geo-partitioning
 
-Create 2 universes (source and target) with same configurations (with same regions and zones) as below:
+You start by creating a source and a target universe with the same configurations (the same regions and zones), as follows:
 
-* **Regions:** EU(Paris), Asia Pacific(Mumbai) and US West(Oregon)
-* **Zones:** eu-west-3a, ap-south-1a and us-west-2a
+* Regions: EU(Paris), Asia Pacific(Mumbai) and US West(Oregon)
+* Zones: eu-west-3a, ap-south-1a and us-west-2a
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:190]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 9]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
+  ```sh
+  ./bin/yb-ctl --rf 3 create --placement_info "aws.us-west-2.us-west-2a,aws.ap-south-1.ap-south-1a,aws.eu-west-3.eu-west-3a"
+  ```
 
-### Steps to create Replication(Unidirectional):
-1. Collect partition table UUIDs from source universe (partition tables, here transactions_eu, transactions_in and transactions_us) - these will be used while setting up replication. To collect table UUIDs, go to the `Tables` section in the Admin UI (127.0.0.1:7000).
+Create tables, table spaces, and partition tables at both the source and target universes, as per the following example:
+
+* Main table: transactions
+* Table spaces: eu_ts, ap_ts and us_ts
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:205]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 10]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
-                  PRIMARY KEY (user_id HASH, account_id, geo_partition))
-                  DEFAULT TABLESPACE us_ts;
-```
+  ```sh
+  ./bin/yb-ctl --rf 3 create --placement_info "aws.us-west-2.us-west-2a,aws.ap-south-1.ap-south-1a,aws.eu-west-3.eu-west-3a"
+  ```
 
-### Steps to create Replication(Unidirectional):
-1. Collect partition table UUIDs from source universe (partition tables, here transactions_eu, transactions_in and transactions_us) - these will be used while setting up replication. To collect table UUIDs, go to the `Tables` section in the Admin UI (127.0.0.1:7000).
+Create tables, table spaces, and partition tables at both the source and target universes, as per the following example:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:202]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 11]
```diff
@@ -158,23 +158,21 @@ For example:
   000030a5000030008000000000004000,000030a5000030008000000000004005,dfef757c415c4b2cacc9315b8acb539a
 ```
 
-### Source and Target Universes have Different Certificates
+When universes use different certificates, you need to store the certificates for the producer universe on the target universe, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:161]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 12]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
-  {{< note title="Note" >}}
-  Only the partition tables need to be specified for replication setup - it is not necessary to specify the parent table.
-  {{< /note >}}
+<br>
 
-2. Run the replication setup command for the source universe:
+2. Run the replication setup command for the source universe, as follows:
     ```sh
     ./bin/yb-admin -master_addresses <consumer_master_addresses> \
     setup_universe_replication <producer universe UUID>_<replication_stream_name> \
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:259]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 13]
```diff
@@ -184,255 +182,266 @@ When both universes use different certificates, you need to store the certificat
...
     ```
-3. Observe replication setup(`xClusterSetup1`) in the YugabyteDB Anywhere UI (At Replication tab in source universe and target universe)
+3. Optionally, if you have access to YugabyteDB Anywhere, you can observe the replication setup (`xClusterSetup1`) by navigating to **Replication** on the source and target universe.
 
-## xCluster Setup in K8s ( pod to pod connectivity )
+## Set up asynchronous replication in Kubernetes
 
-1. Create 2 universes (source and target).
-2. Create tables in both source and target universes
+In the Kubernetes environment, you can set up a pod to pod connectivity, as follows:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:274]: It was connectivity here (the section looked like a developer wrote it)

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:280]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:296]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:444]: fixed

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 14]
```diff
@@ -184,255 +182,264 @@ When both universes use different certificates, you need to store the certificat
...
-  fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
-  ```
+      ```sh
+    ./bin/yb-admin-master_addresses 127.0.0.11:7100,127.0.0.12:7100,127.0.0.13:7100 setup_universe_replication \
+      00000000-1111-2222-3333-444444444444_xCluster1 127.0.0.1:7100,127.0.0.2:7100,127.0.0.3:7100 \
+      000033e1000030008000000000004000,000033e1000030008000000000004003,000033e1000030008000000000004006 \
+      fb156717174941008e54fa958e613c10,a2a46f5cbf8446a3a5099b5ceeaac28b,c967967523eb4e03bcc201bb464e0679
+      ```
+
+You can modify the bootstrap as follows:
```
Comment (nspiegelberg) [inline docs/content/preview/deploy/multi-dc/async-replication.md:413]: Should this be a subitem in the page outline?

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 15]
```diff
@@ -184,255 +182,264 @@ When both universes use different certificates, you need to store the certificat
...
+
+- To wipe the test setup, use the `delete_universe_replication` command.
+- After running the `bootstrap_cdc_producer` command, you can verify entries via the `list_cdc_streams` command.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:416]: Changed

File: docs/content/preview/deploy/multi-dc/async-replication.md (hunk 14 above)
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:413]: Added a section header

File: docs/content/preview/deploy/multi-dc/async-replication.md [hunk 16]
```diff
@@ -184,255 +182,264 @@ When both universes use different certificates, you need to store the certificat
...
 
-In the event you cannot stop incoming user traffic, then the recommended approach would be to apply DDLs on the source cluster and use [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster) flow above. A more detailed runbook would be as follows:
-- Stop replication, in advance of DDL changes.
-- Apply all your DDL changes to the source cluster.
-- Take a backup of the source cluster, of all the relevant tables that you intend to replicate changes for.
-  - Make sure to use the [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster) flow, as described above.
-- Restore this backup on the sink cluster.
-- [Setup replication](../../../admin/yb-admin/#setup-universe-replication) again, for all of the relevant tables.
-  - Make sure to pass in the `bootstrap_ids`, as described above.
+If you cannot stop incoming user traffic, then the recommended approach would be to apply DDLs on the source universe combined with bootstrapping. Specifically, you would need to do the following:
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/deploy/multi-dc/async-replication.md:439]: Changed

**Pull request merged**